src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1100,7 +1100,7 @@ private void ScheduleBackoffTimer()
             else
             {
                 _smallBackoffCount += 1;
-                var backoffDeadlineNanoTime = TimeSpan.FromTicks((MonotonicClock.GetNanos() + _adaptiveBackoffNanos).ToTicks());
```

Comment:
So this was a terrible bug - anytime we had to do a "small" backoff the estimate backoff duration included **the total uptime of the machine**, which would effectively kill the connection.